Runtime/Impulse/CinemachineImpulseSource.cs

```diff
@@ -16,6 +16,12 @@ namespace Cinemachine
     [SaveDuringPlay]
     public class CinemachineImpulseSource : MonoBehaviour
     {
+#if UNITY_EDITOR
+        [NoSaveDuringPlay]
+        [Tooltip("(Editor only, while in Play mode) Click this to generate an impulse right now")]
+        public bool TestNow;
```

Comment:
I could not make this work. Clicked and nothing happened (tried in play mode).

Reply:
removed it